packages/react/src/dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,26 @@
+import { type ComponentPropsWithRef, type ReactNode, forwardRef } from "react";
+
+import type { ExtendProps } from "../utils";
+
+import { Box } from "../box";
+import * as styles from "./Dropdown.css";
+
+type DropdownProps = ExtendProps<
+  ComponentPropsWithRef<typeof Box>,
+  {
+    children?: ReactNode;
+    rightSection?: ReactNode;
+  }
+>;
+
+export const Dropdown = forwardRef<HTMLDivElement, DropdownProps>(
```

Comment:
let's call this `Menu` to prevent confusion
```suggestion
export const Menu = forwardRef<HTMLDivElement, MenuProps>(
```

https://open-ui.org/components/menu.research/